lms/models/lti_user.py

```diff
@@ -28,6 +37,9 @@ class LTIUser:  # pylint: disable=too-many-instance-attributes
     application_instance_id: int
     """ID of the application instance this user belongs to"""
 
+    lti: LTI
```

Comment:
This is a bit nonsensical at face value, lti_user should be a member of lti no the other way around.

I've tried a few version of what we discussed earlier during the week and couldn't find a path to a refactor that was not massive and touched many parts of the app in non trivial ways.

The path to that refactor is changing LTIUser into "SecureSession" "SecureToken" or similar. I reckon this change goes into that direction.

Reply:
I think it would be a very good idea to put in some comments about the intended future design of this now. I think it's quite likely that we'll come back to this in the future having forgotten what we were thinking. At that point we can save ourselves the "We must have wanted it this way for a good reason" chat.